components/ai_chat/resources/page/constants/file_types.ts

```diff
@@ -0,0 +1,19 @@
+/* Copyright (c) 2025 The Brave Authors. All rights reserved.
+ * This Source Code Form is subject to the terms of the Mozilla Public
+ * License, v. 2.0. If a copy of the MPL was not distributed with this file,
+ * You can obtain one at https://mozilla.org/MPL/2.0/. */
+
+// Supported image file types for AI Chat uploads (matches C++ backend)
+export const SUPPORTED_IMAGE_TYPES = [
+  'image/png',
+  'image/jpeg',
+  'image/jpg',
+  'image/webp'
+] as const
```

Comment:
Extremely optional: I wonder if its worth defining these as mojom constants? I guess possibly not if we need to put them in an array anyway (and it looks like on desktop we just go off file extension anyway)